### ISSUE TITLE
Search field dropdown

### ISSUE THREAD

**Issue by fenhl** [taw/magic-search-engine#4]
This looks great! One thing that would be nice to have is a list of matching cards when using the search field, like [magidex.com](https://magidex.com/) shows.

**Comment by taw** [comment]
Yeah, that would useful to add. It needs some care so it doesn't hammer the server with one-query-per-character and wrt presentation, but these are solvable problems.
